org.open_scap.scap_workbench.appdata.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<component type="desktop">
-    <id>org.openscap.scap_workbench.desktop</id>
+<component type="desktop-application">
+    <id>org.open_scap.scap_workbench</id>
     <metadata_license>CC0-1.0</metadata_license>
     <project_license>GPL-3.0+</project_license>
     <name>SCAP Workbench</name>
     <summary>A GUI tool that serves as an SCAP scanner and provides tailoring functionality for SCAP content</summary>
+    <launchable type="desktop-id">org.open_scap.scap_workbench.desktop</launchable>
```

Comment:
What about
```suggestion
    <launchable type="desktop-id">org.open_scap.scap_workbench.desktop</launchable>
    <content_rating/>
```

Reply:
But what rating exactly? We are using downloadable content, so we can't know what could possibly be there.

Reply:
It is true that workbench will display whatever the user supplies to it, and I can imagine `money-advertising` becoming an issue in the extreme realistic case, but with this approach you couldn't rate a text editor or media player.
Anyway, I don't want to get stuck here because of a validation info output.

Reply:
That's pretty much the reason I left it unrated. It's better to not say anything than say something controversial.

Reply:
And we, unlike a media player, supply a list of downloadable content, which ATM is under our control, but still technically not part of the package.